src/api/status/public/js/posts-count.js

```diff
@@ -0,0 +1,18 @@
+const totalPostsEl = document.getElementById('totalPosts');
+const fetchPostsCount = () =>
+  fetch('https://dev.api.telescope.cdot.systems/v1/posts').then((res) =>
+    res.headers.get('x-total-count')
+  );
+
+const getPostsCount = async () => {
+  try {
+    const total = await fetchPostsCount();
+    totalPostsEl.innerText = total ?? 'N/A';
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+window.onload = function () {
```

Comment:
In public/script.js, `window.onload` is already assigned. Call your function in there.

https://github.com/Seneca-CDOT/telescope/blob/00213c8cd1e9020196dcc892772fc04f4f0fdc23/src/api/status/public/script.js#L51-L53

Reply:
I don't think we need to call it on `window.onload` instead we can call it in its separate file and have the file included in the `index.html`

Reply:
Either way it needs to wait on the DOM being ready.